types/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// TypeScript Version: 3.7
+// TypeScript Version: 4.0
```

Comment:
:thinking: I'm not quite sure how to handle this SemVer wise.
On one hand this should still be compatible with 3.7, your changes are non-breaking.
But on the other hand to use the fragment feature in the tests, this is updated to 4.0, meaning we no longer test previous version.
Thoughts on whether this should be patch or major?

Reply:
Considering this doesn't break functionality for older TypeScript versions, and the fact I haven't had any issues upgrading to TypeScript 4.0 in other projects, I'd say this is semver patch.

The alternative is not to test fragments, which I think is worse.

This could be combined. The types could be added, but the fragment tests could be delayed for a semver major release. Along with the minimal TypeScript version bump.

Reply:
If the code that runs doesn’t change, the semver version doesn’t change, imo.

I’m fine with:
- fragments in tests, updating the version string there (maybe describing that it’s only for tests), releasing it in a patch
- for now commented out fragments in tests, not updating the version string there, releasing it in a patch

Reply:
IMO this can be released as-is then. :smile: